core/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.grappus.core">
+
+    <application
+        android:label="@string/app_name"
+        android:supportsRtl="true">
```

Comment:
Was this explicitly required?

Reply:
Nope, but a good practice as it may show warning in xml accordingly.